frontend/src/lib/components/common/form/Field.svelte

```diff
@@ -11,21 +13,27 @@ Meant to be mainly used with two chidlren that are aligned on both ends of the f
 ### Properties
 
 - `customStyle`: A custom style passed on to the parent component.
-
-You should not try to use a variant and customize at the same time.
+- `bgColor`: The background color of the field.
+- `id`: The id of the field used for labeling. Optional.
```

Comment:
The id of the `input` which the `label` is for. You must supply either both `id` and `label` or neither of them. Optional.